### PR TITLE
{bio}[intel/2018a] HH-suite v3.0-beta.3

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/soedinglab/hh-suite/archive/',
+    'https://github.com/soedinglab/ffindex_soedinglab/archive/',
+]
+sources = [
+    'v%(version)s.tar.gz',
+    '%s.tar.gz' % ffindex_commit,
```
Comment (boegel) [inline easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb]: @smoors Please rename on download (I'd use a datestamp of that commit), cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    # 74550c8bde3d5b450755ec4be5e9cd56f28a231b.tar.gz
+    '3978901196ee9d34cfb1a847cf27e224d104957f1c6675d900c823a8ee78b1d9',
+]
+
+builddependencies = [('CMake', '3.10.2')]
+
+preconfigopts = 'cp -a ../ffindex_soedinglab-%s/* ' % ffindex_commit
+preconfigopts += '%(builddir)s/%(namelower)s-%(version)s/lib/ffindex && '
+
+configopts = '-DCMAKE_BUILD_TYPE=RelWithDebInfo -G "Unix Makefiles" '
```
Comment (boegel) [inline easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb]: @smoors Why with debug enabled? We usually og with `Release`...

Comment (smoors) [reply easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb]: @boegel it's recommended by the developers, but "I don't strongly oppose this change" (citing a famous developer :)

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    'https://github.com/soedinglab/hh-suite/archive/',
+    'https://github.com/soedinglab/ffindex_soedinglab/archive/',
+]
+sources = [
+    'v%(version)s.tar.gz',
+    {'download_filename': '%s.tar.gz' % ffindex_commit, 'filename': 'ffindex-20171201.tar.gz'},
+]
+checksums = [
+    '483039a642fba375e3ba6ee49e38c16695dfa4f88cad23b09cd042755db01c12',  # v3.0-beta.3.tar.gz
+    # 74550c8bde3d5b450755ec4be5e9cd56f28a231b.tar.gz
```
Comment (boegel) [inline easybuild/easyconfigs/h/HH-suite/HH-suite-3.0-beta.3-intel-2018a.eb]: @smoors comment doesn't match actual filename, should be `# ffindex-20171201.tar.gz` (and can then be a single line again), `--inject-checksums` will fix this for you

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2533.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d2129989e4891c5b73475761068be8c3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ddaa7b2bf2a2733fc829d0f0c9c707c9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**